pkg/sql/rowexec/project_set.go

```diff
@@ -125,7 +125,15 @@ func newProjectSetProcessor(
 // MustBeStreaming implements the execinfra.Processor interface.
 func (ps *projectSetProcessor) MustBeStreaming() bool {
 	// If we have a single streaming generator, then the processor is such too.
-	for _, gen := range ps.gens {
+	for _, fn := range ps.funcs {
+		if fn == nil {
+			continue
+		}
+		gen, err := ps.generatorForFuncExpr(fn)
```

Comment:
It's a bummer to do this here, because I think that constructing this generator actually does work in some cases.